### DISCUSSION TITLE
More, custom heading types in inline toolbar dropdown

### DISCUSSION THREAD

**Discussion by abc013** [TriliumNext/Trilium#4252] [Q&A] [answered]
Hey there, I am looking for a way to add more variety to my note texts, using `css` classes specifically. The problem is that raw HTML, as a far as I know, can't just be inserted into a note.
However, I stumbled across the [CKeditors wiki page about headings](https://ckeditor.com/docs/ckeditor5/latest/features/headings.html) and this is kind of exactly what I need: Some more heading templates to configure my text with.

Is there any way to add more headings to the dropdown without altering trilium source code, i.e. via scripting?

Thanks a lot!

**Comment by sottey** [comment, marked as answer]
I looked into this yesterday. I don't believe there is. I was hoping I could somehow add in Code Folding but it would take a recompile as far as I can tell.

Could it be done with scripting? In theory. Having a listener which alters a text's display. But this seems super heavy handed.

I know there is some talk about possible plugin capability coming to Trilium someday. This might be a good candidate.

**Reply by abc013** [reply]
Ah, thats too bad to hear. But i hope it will get added in the future some time soon. Thanks for your research!
